packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -111,18 +111,32 @@ exports.onCreateNode = ({ node, getNode, actions }, pluginOptions) => {
 // https://www.gatsbyjs.org/docs/mdx/programmatically-creating-pages/#create-pages-from-sourced-mdx-files
 exports.createPages = async ({ graphql, actions, reporter }) => {
   const allMdxPagesResult = await graphql(`
+    fragment fieldsFragment on Mdx {
+      id
+      fields {
+        slug
+        title
+        beta
+        isGlobalBeta
+        excludeFromSearchIndex
+      }
```

Comment:
PS: I don't think the content pages and the release note pages have the same frontmatter fields (for example the beta flags).
Therefore, I would not use a fragment and explicitly list the fields on each query.

Maybe @nkuehn can give some more input?

Reply:
1. yes, the meaning and choice of allowed frontmatter are completely different and I would be happy if the query prevents using e.g. a release notes frontmatter (like the "slug" which is there only for migration) in regular pages. 
2. most of the frontmatter defined in the release note spec example is not queried here yet.